### PR TITLE
VIH-7409 sds prep

### PR DESCRIPTION
**Pull request by surfsquid**
Migrated to the new client nuget packages, added health check endpoint and docker SDS prep 
https://hmctsreform.visualstudio.com/VirtualHearings/_build/results?buildId=30345&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by surfsquid — COMMENTED** [review]
looks fab, IMHO

File: SchedulerJobs/SchedulerJobs/Startup.cs [hunk 1]
```diff
@@ -51,14 +54,35 @@ public static void ConfigureServices(IServiceCollection services)
...
 
-            services.AddHttpClient<IUserApiService, UserApiService>()
-                .AddHttpMessageHandler<UserServiceTokenHandler>();
+            services.AddHttpClient<IUserApiClient, UserApiClient>()
+                .AddHttpMessageHandler<UserServiceTokenHandler>()
+                .AddTypedClient(httpClient =>
+                {
+                    var client = UserApiClient.GetClient(httpClient);
+                    client.BaseUrl = hearingServicesConfiguration.UserApiUrl;
+                    client.ReadResponseAsString = true;
```
Comment (levibutler) [inline SchedulerJobs/SchedulerJobs/Startup.cs:83]: not needed now ;) ?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1,36 +1,163 @@
...
+        - group: vh-apps-settings-generic
+        - group: vh-preview
+        - group: vh-scheduler-jobs
+      coverletCoverageExclusions:
+        - "[*]SchedulerJobs.Common.*"
+        - "[*]SchedulerJobs.UnitTests.*"
+        - "[Testing.Common]*"
+      sonarCloud_ExtraProperties: |
+        sonar.cs.opencover.reportsPaths=$(Common.TestResultsDirectory)\Coverage\coverage.opencover.xml
+        sonar.coverage.exclusions=**/VideoApiService.cs, **/Startup.cs, **/Testing.Common/**, **/SchedulerJobs.Common/ApiHelper/**, **/SchedulerJobs.Common/Helper/Configuration/**, **/SchedulerJobs.Common/Helper/Security/**
```
Comment (levibutler) [inline azure-pipelines.yml:137]: removed

**Review by levibutler — APPROVED** [review]
Nice!

**Pull request merged**